src/main/java/io/github/thebusybiscuit/slimefun4/utils/NumberUtils.java

```diff
@@ -171,9 +170,29 @@ private NumberUtils() {}
         }
     }
 
-    public static @Nonnull String getTimeLeft(int seconds) {
+    public static @Nonnull String getTimeLeft(int ticksLeft) {
         String timeleft = "";
 
+        /**
+         * This is the amount calculated in addProgress(). 
+         * This will be the scale used for timer.
+         */
+        double timeScale = Slimefun.getCfg().getInt("URID.custom-ticker-delay") / 10.0D;
+        double normalScale = timeScale - Math.round(timeScale);
+
+        int seconds = 0;
+
+        if (normalScale < 0) {
+            //Adjust for positive change
+            seconds = (int) (ticksLeft * (normalScale / 2 + 1.0D));
+        } else if (normalScale > 0) {
+            // Adjust for negative change 
+            seconds = (int) (ticksLeft * (normalScale + 1.0D));
+        } else {
+            // No change 
+            seconds = ticksLeft;
+        }
```

Comment:
Not a necessary change but we can get rid of the else bracket by defaulting seconds to ticksLeft

```suggestion
        int seconds = ticksLeft;

        if (normalScale < 0) {
            //Adjust for positive change
            seconds = (int) (ticksLeft * (normalScale / 2 + 1.0D));
        } else if (normalScale > 0) {
            // Adjust for negative change 
            seconds = (int) (ticksLeft * (normalScale + 1.0D));
        }
```

Reply:
Good idea yes.